front/pages/w/[wId]/index.tsx

```diff
@@ -18,27 +19,32 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     };
   }
 
+  const isOnV2 = isOnAssistantV2(owner);
+  const redirectRoute = isOnV2
+    ? `/w/${context.query.wId}/assistant/new`
+    : `/w/${context.query.wId}/u/chat`;
+
   if (owner.role === "user") {
```

Comment:
we can remove that whole if block right?

Reply:
Indeed! Done